tests/test_pluggable.py

```diff
@@ -316,6 +380,39 @@ def test_retrieve_subject_token_oidc_jwt(self):
 
             assert subject_token == self.EXECUTABLE_OIDC_TOKEN
 
+    @mock.patch.dict(os.environ, {"GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES": "1"})
+    def test_retrieve_subject_token_oidc_jwt_interactive_mode(self, tmpdir):
```

Comment:
You can probably just merge this with the test above using a for loop (loop over output file responses).

Reply:
Refactoring to have all successful paths together will non-interactive and interactive together with all types of supported token type.